src/bun.js/bindings/BunDebugger.cpp

```diff
@@ -120,10 +124,10 @@ class BunInspectorConnection : public Inspector::FrontendChannel {
             };
         }
 
-        if (waitingForConnection) {
-            waitingForConnection = false;
-            Debugger__didConnect();
-        }
+        // if (waitingForConnection) {
+        // waitingForConnection = false;
+        // Debugger__didConnect();
+        // }
```

Comment:
I'm still not sure about the importance of this, so left them in for review and then can remove once its determined to work well.